ratpack-core/src/test/groovy/ratpack/core/http/client/HttpClientRedirectionSpec.groovy

```diff
@@ -182,12 +182,14 @@ class HttpClientRedirectionSpec extends BaseHttpClientSpec {
     }
 
     then:
-    text == "http://ratpack.io=10"
+    text == "${URLEncoder.encode("http://ratpack.io=10", "UTF-8")}"
```

Comment:
This doesn't look right to me.  A query param with a single level of encoding, when accessed through `request.queryParams`, should be decoded.  This test makes it look like that has been broken.

Reply:
We may need more test coverage to determine if other scenarios are working as intended or not.

Reply:
Agreed, definitely will need more test coverage.  I'll put one in that mimics/illustrates the Develocity issue more clearly.

Reply:
I backed the assertion out as it appears to be desirable for query parameters to be converted. I did, however, add some additional tests, along with some altered functionality.